Clients/src/presentation/components/ProjectCard/ProgressBar/index.tsx

```
@@ -0,0 +1,36 @@
+
+import { Box } from "@mui/material";
+import { FC } from "react";
+
+interface ProgressBarProps {
+    progress: string
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({progress}) => {
+    const progressCount = (progressString: string): number => {
+        const [completed, total] = progressString.split('/').map(Number);
+        if (isNaN(completed) || isNaN(total) || total === 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)